src/Diagnostics.RuntimeHost/Diagnostics.RuntimeHost.csproj

```diff
@@ -48,4 +48,8 @@
     </Content>
   </ItemGroup>
 
+  <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
+    <Exec Command="taskkill /f /fi &quot;WINDOWTITLE eq Diagnostics.RuntimeHost&quot;" />
```

Comment:
Its not a big deal but when I run the project, it just somehow reuses the same process so not sure why we might need this.

Reply:
I have reverted this change.